Content.Client/Ghost/Roles/UI/GhostRoleRulesWindow.xaml.cs

```diff
@@ -20,19 +20,5 @@ public GhostRoleRulesWindow(string rules, Action<BaseButton.ButtonEventArgs> req
             TopBanner.SetMessage(FormattedMessage.FromMarkupPermissive(rules + "\n" + Loc.GetString("ghost-roles-window-rules-footer")));
```

Comment:
remove the ` + "\n" + Loc.GetString("ghost-roles-window-rules-footer")`

Reply:
and in ghost-gui.ftl you can remove the `ghost-roles-window-rules-footer`

Reply:
a little workaround since it's now a cvar. it now says "after a few seconds" instead of "after 5 seconds"